multimedia/x264/Portfile

```diff
@@ -32,9 +32,12 @@ minimum_xcodeversions {9 3.1}
 
 depends_build       port:nasm
 
+
 # as of 20190313 the PPC assembly uses VSX, which is Power7+ only. We could disable asm, but
 # instead we can use the previous ppc assembly,  which does compile and provides the same functions
-patchfiles-append   patch-x264-older-ppc-code.diff
+
+# -- this seems to not be available from the distfiles mirrors anymore and is PPC only?
+#patchfiles-append   patch-x264-older-ppc-code.diff
```

Comment:
I don't know if this is still important for any ongoing PPC support, but I was unable to build the port from source as the distfiles for this patch couldn't be downloaded.

Reply:
The file should be in `multimedia/x264/files/`, eg:
```
$ ls -la /opt/local/var/macports/sources/rsync.macports.org/release/tarballs/ports/multimedia/x264/files
total 16
drwxr-xr-x  3 root  pulse    96 30 Mar  2019 .
drwxr-xr-x  4 root  pulse   128 30 Mar  2019 ..
-rw-r--r--  1 root  pulse  7125 30 Mar  2019 patch-x264-older-ppc-code.diff
```

perhaps you are not dealing with a full repo?